services/venice-controller/src/main/java/com/linkedin/venice/controller/kafka/consumer/AdminExecutionTask.java

```diff
@@ -622,18 +625,30 @@ private void handleAddVersion(AddVersion message) {
             replicationMetadataVersionId);
       }
     } else {
-      // New version for regular Venice store.
-      admin.addVersionAndStartIngestion(
-          clusterName,
-          storeName,
-          pushJobId,
-          versionNumber,
-          numberOfPartitions,
-          pushType,
-          remoteKafkaBootstrapServers,
-          rewindTimeInSecondsOverride,
-          replicationMetadataVersionId,
-          message.versionSwapDeferred);
+      boolean skipConsumption = message.targetedRegions != null && !message.targetedRegions.isEmpty()
```

Comment:
can use string utils and RegionUtils methods here

Reply:
`targetedRegions` is a list of String so these methods don't apply. We can keep it this way.